platform_versioned_docs/version-24.1/enterprise/configuration/configtables/data_features_yml.yml

```diff
@@ -0,0 +1,11 @@
+---
+-
+  tower.yml:            '`tower.content.max-file-size`'
+  Description: >
+    You can set the Data Explorer downloadable file size limit. Increasing this value may degrade Platform performance.
```

Comment:
Curious as to whether this has a maximum limit?

Reply:
nope, there is no enforcing of a maximum limit
so basically customers are allowed to set a really high value and experience super slow downloads for these big files